draft-packages/form/docs/ToggleSwitchField.stories.tsx

```diff
@@ -72,191 +66,131 @@ export const OnKaizenSiteDemo = () => (
           toggledStatus={toggledStatus}
           onToggle={toggle}
         />
-      </RtlContainer>
-    )}
-  </ToggleStateContainer>
-)
-
-OnRtl.storyName = "On (RTL)"
-
-export const OffRtl = () => (
-  <ToggleStateContainer initialToggledStatus={ToggledStatus.OFF}>
-    {({ toggledStatus, toggle }) => (
-      <RtlContainer>
+      )}
+    </ToggleStateContainer>
+    <ToggleStateContainer initialToggledStatus={ToggledStatus.ON}>
+      {({ toggledStatus, toggle }) => (
         <ToggleSwitchField
+          id="2"
           labelText="Label"
+          labelPosition="end"
           toggledStatus={toggledStatus}
           onToggle={toggle}
         />
-      </RtlContainer>
-    )}
-  </ToggleStateContainer>
-)
-
-OffRtl.storyName = "Off (RTL)"
-
-export const DisabledOn = () => (
-  <ToggleStateContainer initialToggledStatus={ToggledStatus.ON}>
-    {({ toggledStatus, toggle }) => (
-      <ToggleSwitchField
-        labelText="Label"
-        toggledStatus={toggledStatus}
-        onToggle={toggle}
-        disabled
-      />
-    )}
-  </ToggleStateContainer>
-)
-
-export const DisabledOff = () => (
-  <ToggleStateContainer initialToggledStatus={ToggledStatus.OFF}>
-    {({ toggledStatus, toggle }) => (
-      <ToggleSwitchField
-        labelText="Label"
-        toggledStatus={toggledStatus}
-        onToggle={toggle}
-        disabled
-      />
-    )}
-  </ToggleStateContainer>
-)
-
-export const DisabledOnRtl = () => (
-  <ToggleStateContainer initialToggledStatus={ToggledStatus.ON}>
-    {({ toggledStatus, toggle }) => (
-      <RtlContainer>
+      )}
+    </ToggleStateContainer>
+    <ToggleStateContainer initialToggledStatus={ToggledStatus.ON}>
+      {({ toggledStatus, toggle }) => (
         <ToggleSwitchField
+          id="3"
           labelText="Label"
           toggledStatus={toggledStatus}
           onToggle={toggle}
           disabled
         />
-      </RtlContainer>
-    )}
-  </ToggleStateContainer>
+      )}
+    </ToggleStateContainer>
+  </div>
 )
 
-DisabledOnRtl.storyName = "Disabled On (RTL)"
-
-export const DisabledOffRtl = () => (
-  <ToggleStateContainer initialToggledStatus={ToggledStatus.OFF}>
-    {({ toggledStatus, toggle }) => (
-      <RtlContainer>
+DesignSheet.storyName = "Design Sheet (Default)"
+
+export const DesignSheetSheetReversed = () => (
+  <div
+    style={{
+      display: "grid",
+      gridTemplateColumns: "repeat(3, minmax(200px, 400px))",
+    }}
+  >
+    <div>
+      <Heading variant="heading-5" tag="h2" color="white">
+        Default
+      </Heading>
+    </div>
+    <div>
+      <Heading variant="heading-5" tag="h2" color="white">
+        Label Position End
+      </Heading>
+    </div>
+    <div>
+      <Heading variant="heading-5" tag="h2" color="white">
+        Disabled
+      </Heading>
+    </div>
+    <ToggleStateContainer initialToggledStatus={ToggledStatus.ON}>
+      {({ toggledStatus, toggle }) => (
         <ToggleSwitchField
+          id="1"
```

Comment:
You'll want to change the ids for the Reversed stories. They trigger the Default ones when interacted with :)